t/all-basic-operations.t

```
@@ -222,6 +225,8 @@ sub Strings {
 
 sub Geo {
 
+  return if $redis_version < version->parse('3.2.0');
```

Instead of silently skipping these tests, we should use the Test::More framework for skipping tests explicitly: https://metacpan.org/pod/Test::More#Conditional-tests

---

Will amend